packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../common/common';
+
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+//
+// Office UI Fabric
+// --------------------------------------------------
+// Search reader alert styles
+
+.ms-ScreenReaderAlert {
+    @include ms-u-screenReaderOnly;
+}
+
+.ms-PPP {
```

Comment:
What is this class? I don't see it referenced anywhere in the component template.